lec20-publishing.Rmd

```diff
@@ -1,4 +1,261 @@
 ---
 title: "Preparing a scientific item for academic publishing"
 ---
+## Lesson preamble
 
+> ### Lesson objectives:
+>
+> - Understand the roles and responsibilities surrounding authorship
+> - Learn about current conventions around academic publishing and how they
+>   are changing with open access journals and preprint servers
+>
+> ### Lesson outline:
+>
+> - Authorship (25 min)
+>     - Different types of contributors (10 min)
+>     - How to credit data and software creators? 10 min)
+>     - ORCID for keeping track of a scientist's contributions (5 min)
+> - Academic Publishing (40 min)
+>     - An overview of the peer review system (10 min)
+>     - How do you decide where to publish? (10 min)
+>     - Moving away from impact factors (10 min)
+>     - Video: Open Access Explained! (10 min)
+> - Submitting a Scientific Publication (35 min)
+>     - General format (10 min)
+>     - Journal specifications (10 min)
+>     - I've submitted, now what? (5 min)
+>     - General writing tips (10 min)
+----
+
+*The following has been adapted from [Lecture 3 of the original RQM
+course](https://cbahlai.github.io/rqm-template/lessons/week3/)*
+
+## Authorship
+
+### Different Types of Contributors
+
+What kind of contributions make someone an author of a scientific publication?
+Here are some common types of contributions:
```

Comment:
Might need to mention that *usually* authorship requires having at least two of these plus having read the manuscript and approving it's content.

Maybe link to this [page](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html). Many biomedical journals use this recommendation to define "authorship".